selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/SystemAdminOrganizationViewsTest.java

```diff
@@ -24,151 +25,134 @@
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 /**
- * Test validates organization views for admin of the organization.
+ * Test validates organization views for system admin.
  *
  * @author Ann Shumilova
  */
-public class AdminOfParentOrganizationTest {
-
-  private static final Logger LOG = LoggerFactory.getLogger(AdminOfParentOrganizationTest.class);
+public class SystemAdminOrganizationViewsTest {
+  private static final String PARENT_ORG_NAME = generate("organization", 5);
+  private static final String CHILD_ORG_NAME = generate("organization", 5);
 
   private OrganizationDto parentOrganization;
   private OrganizationDto childOrganization;
 
-  @Inject private OrganizationListPage organizationListPage;
-  @Inject private OrganizationPage organizationPage;
-  @Inject private NavigationBar navigationBar;
-  @Inject private CheMultiuserAdminDashboard dashboard;
-  @Inject private TestIdeUrlProvider testIdeUrlProvider;
-  @Inject private TestDashboardUrlProvider testDashboardUrlProvider;
-  @Inject private TestUser testUser;
-
   @Inject
   @Named("admin")
   private TestOrganizationServiceClient testOrganizationServiceClient;
 
+  @Inject private OrganizationListPage organizationListPage;
+  @Inject private OrganizationPage organizationPage;
+  @Inject private NavigationBar navigationBar;
+  @Inject private CheMultiuserAdminDashboard dashboard;
   @Inject private AdminTestUser adminTestUser;
 
   @BeforeClass
   public void setUp() throws Exception {
-    testOrganizationServiceClient.create(NameGenerator.generate("organization", 5));
-    parentOrganization =
-        testOrganizationServiceClient.create(NameGenerator.generate("organization", 5));
+    parentOrganization = testOrganizationServiceClient.create(PARENT_ORG_NAME);
     childOrganization =
-        testOrganizationServiceClient.create(
-            NameGenerator.generate("organization", 5), parentOrganization.getId());
+        testOrganizationServiceClient.create(CHILD_ORG_NAME, parentOrganization.getId());
+    testOrganizationServiceClient.addAdmin(parentOrganization.getId(), adminTestUser.getId());
 
-    testOrganizationServiceClient.addAdmin(parentOrganization.getId(), testUser.getId());
-    testOrganizationServiceClient.addMember(childOrganization.getId(), testUser.getId());
-    dashboard.open(testUser.getName(), testUser.getPassword());
+    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
   }
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteById(childOrganization.getId());
-    testOrganizationServiceClient.deleteById(parentOrganization.getId());
+    for (OrganizationDto organization : testOrganizationServiceClient.getAll())
+      testOrganizationServiceClient.deleteById(organization.getId());
```

Comment:
Such code can affect organizations which are created in parallel threads of test execution.